### ISSUE TITLE
"mocha.options" are not passed to mocha when test is run in debug mode

### ISSUE THREAD

**Issue by abeninskibede** [maty21/mocha-sidebar#93]
"mocha.options" are not passed to mocha when test is run in debug mode resulting in timeout because I cannot overwrite mocha timeout of 2000 ms.

#### Steps to Reproduce
1. Put timeout in mocha options. 
``` 
"mocha.options": {
    "timeout": 100000
  } 
```
2. Put a breakpoint in the beforeEach and in the test. 
3. Run test in debug mode.
4. Wait for a while in beforeEach function of the test.
5. Continue and observe timeout error.

#### Expected Results
options are passed to mocha in debug mode

#### Actual Results
options are not passed to mocha in debug mode
```nodejs\node.exe --inspect-brk=17312 User\extensions\maty.vscode-mocha-sidebar-0.17.5\node_modules\mocha\bin\_mocha ./App/**/**.test.js -r babel-register ```
#### Settings file

```
{
  "mocha.requires": [
    "babel-register"
  ],
  "mocha.files.glob": "./**/App/**/**.test.js",
  "mocha.logVerbose": true,
  "mocha.options": {
    "timeout": 100000
  }
}
```

### Versions

- VScode: v1.22.2
- Mocha SideBar: v0.17.1
- Node Version: v9.6.1

### OS version

- [ ] Linux 
- [ ] Mac
- [x] Windows

**Comment by yehiyam** [comment]
fixed in 0.17.7

**Comment by abeninskibede** [comment]
Now when I run debug with the new verison I get: 
![image](https://user-images.githubusercontent.com/8179559/39127775-aed59b14-470e-11e8-8b5a-0cd607cd960a.png)

It was perfectly fine before the update.

**Comment by yehiyam** [comment]
I'll investigate

**Comment by yehiyam** [comment]
fixed in 0.17.8
@abeninskibede could you please verify

**Comment by abeninskibede** [comment]
Hey,

It works fine.

Thank you for the quick fixes :)
